cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/namespace/StorageProviderNamespaceAdminTest.java

```diff
@@ -144,7 +144,7 @@ public void testNSWithCustomLocation() throws Exception {
       file1.exists()));
     // but custom namespace location should still exists
     Assert.assertTrue(custom.exists());
-    custom.delete();
+    Assert.assertTrue(custom.delete());
```

Comment:
Question: Should we fail the test case itself if we fail to delete a temp directory in cleanup ? This deletion is not a part of code but rather a cleanup in test.